src/index.js

```diff
@@ -18,4 +21,11 @@ function render(ui, {container = document.createElement('div')} = {}) {
   }
 }
 
-export {render, Simulate, wait}
+// fallback to synthetic events for DOM events that React doesn't handle
+;['change', 'select', 'mouseEnter', 'mouseLeave'].forEach(eventName => {
```

Comment:
I prefer to give names to arrays rather than doing this.

```javascript
const syntheticEvents = ['change', 'select', 'mouseEnter', 'mouseLeave']
syntheticEvents.forEach(eventName => {
  document.addEventListener(eventName.toLowerCase(), e => {
    Simulate[eventName](e.target, e)
  })
})
```

Notice that I used `document` rather than `window`. I'm not certain whether that makes much of a difference in practice, but I checked the React source and I'm pretty sure that's where they attach event handlers.